### PR TITLE
Works section

### PR DESCRIPTION
**Pull request by JhonatanErosa**
*Added new branch
*Added html elements
*Added style.css for new html elements
*Fixed linter errors
*Update README.md file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,29 +8,180 @@
...
+                    <a href="" class="btn">HTML</a>
+                    <a href="" class="btn">CSS</a>
+                    <a href="" class="btn">JavaScript</a>
+                </div>
+                <div class="project-link">
+                    <a href="" class="btn2">See Project</a>
+                </div>
+            </div>
+        </div>
+    </div>
```
Comment (mahbubul14) [inline index.html]: - [x] The container of the section should be wrapped in a `section` tag. The `section` tag in HTML is used to group together related content on a webpage. It helps in organizing and structuring the content in a meaningful way. :+1:

File: index.html [hunk 2]
```diff
@@ -8,29 +8,180 @@
...
+            </div>
+            <div class="frameworks-menu">
+                <h3>Frameworks</h3>
+                <img src="src/img/icons/chevron-right-regular-24.png" alt="des 2">
+            </div>
+            <div class="skills-menu">
+                <h3>Skills</h3>
+                <img src="src/img/icons/chevron-right-regular-24.png" alt="des 2">
+            </div>
         </div>
```
Comment (mahbubul14) [inline index.html]: - [x]  The container of the section should be wrapped in a section tag. The `section` tag in HTML is used to group together related content on a webpage. It helps in organizing and structuring the content in a meaningful way. :+1:

File: index.html [hunk 3]
```diff
@@ -8,29 +8,180 @@
...
+                <div class="project-link">
+                    <a href="" class="btn2">See Project</a>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="about-section">
+        <section class="cont-about">
+            <div class="about-myself">
+                <h1 class="about-title">About Myself</h1>
```
Comment (mahbubul14) [inline index.html]: - [x]  Kindly change this `h1` to any other hx (like `h2`, `h3` etc)  tag. It is better for your page to have just one h1 tag, this makes your website semantic and easily readable by browsers. :+1:

File: index.html [hunk 4]
```diff
@@ -8,29 +8,180 @@
...
+                <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            </div>
+            <div class="connect">
+                <p>LET'S CONNECT</p>
```
Comment (mahbubul14) [inline index.html]: - [x]  Please consider changing the `p` tag to an `hx` tag (such as `h2`, `h3`, etc.) for your page's titles. It is advisable to use `hx` tags as they are more suitable for organizing the content on your webpage. :+1:

File: index.html (hunk 2 above)
Comment (mahbubul14) [inline index.html]: - [x] As the project requirements. please use `ul` and `li` for categories in "About myself" section. Languages, Frameworks, and Skills should be list items. :+1:

File: index.html [hunk 5]
```diff
@@ -8,29 +8,180 @@
...
+                    <li class="c-text"><h4> 2015</h5></li>
+                </ul>
+                <div class="card-text">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="languages">
+                    <a href="" class="btn">HTML</a>
+                    <a href="" class="btn">CSS</a>
+                    <a href="" class="btn">JavaScript</a>
+                </div>
```
Comment (mahbubul14) [inline index.html]: - [x]  As the project requirements, please use `ul` and `li` for categories in "Work" section. Html, CSS, and JavaScript should be list items. :+1:

File: index.html [hunk 6]
```diff
@@ -8,29 +8,180 @@
...
+                    </div>
+                    <div class="html-box">
+                        <img src="src/img/icons/Ellipse 1 (1).png" alt="HTML-Icon">
+                        <p>HTML</p>
+                    </div>
+                    <div class="css-box">
+                        <img src="src/img/icons/Ellipse 1 (2).png" alt="CSS-Icon">
+                        <p>CSS</p>
+                    </div>
+                </div>
```
Comment (mahbubul14) [inline index.html]: - [x] The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`. Kindly do the change. :+1:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @JhonatanErosa,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.
:heavy_check_mark: well-implemented figma design.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [x] Kindly try to write more commit messages. Committing often keeps your commits small and, again, helps you commit only related changes. Moreover, it allows you to share your code more frequently with others. That way it's easier for everyone to integrate changes regularly and avoid having merge conflicts.

- [x] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "Fixed bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html [hunk 7]
```diff
@@ -8,28 +8,179 @@
...
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <ul class="languages">
+                    <li><a href="" class="btn">HTML</a></li>
+                    <li><a href="" class="btn">CSS</a></li>
+                    <li><a href="" class="btn">JavaScript</a></li>
+                </ul>
+                <div class="project-link">
+                    <a href="" class="btn2">See Project</a>
+                </div>
```
Comment (AbdusSattar-70) [inline index.html:131]: [ OPTIONAL ] - [ ]

- [ ]  To ensure consistency with the Figma design, it is crucial to align the button interaction style accordingly. 🥱🥱

- [ ] It appears that your current button style does not match the Figma design. As per the project requirements, it is essential to strictly adhere to the specified button interaction style from the Figma design. 
- [ ] Please make the necessary adjustments to ensure that your button style strictly follows the Figma design, maintaining visual coherence and meeting the project requirements.
- [ ] Figma 
Screenshot 2023-06-03 200911

**Review by AbdusSattar-70 — APPROVED** [review]
### STATUS: APPROVED 🏆 👍 
Hi @JhonatanErosa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- DESCRIPTIVE PR TITLE AND DESCRIPTION
- NICE LOOKING UI 
- CLEAN CODE 
ChelseaMasonApproveGIF

## Optional suggestions:
## Please see the inline comment below: IMPORTANT  🔴 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**